moniker.go

```diff
@@ -7,10 +7,10 @@ import (
 )
 
 // New returns a generic namer using the default word lists.
-func New() Namer {
+func New(nlist []string, dlist []string) Namer {
```

Comment:
Instead of changing the signature of `New()`, could you add a `NewWith([]string, []string)` function? That would be useful in the future, but won't break the current implementations.